api_tests/lib/actor.ts

```diff
@@ -38,55 +26,36 @@ const MOVE_CURSOR_UP_ONE_LINE = "\x1b[1A";
 export class Actor {
     public name: string;
     public wallet: Wallet;
-    private readonly host: string;
-    private cndConfig: CndConfig;
+    private cndConfig: E2ETestActorConfig;
     private readonly declineConfig?: DeclineConfig;
     private readonly sirenActionAutofillParams?: SirenActionAutofillParams;
 
     constructor(
-        name: string,
-        testConfig?: TestConfig,
-        root?: string,
+        name: "alice" | "bob" | "charlie" | "david",
         walletConfig?: WalletConfig,
         declineConfig?: DeclineConfig,
         sirenActionAutofillParams?: SirenActionAutofillParams
     ) {
         this.name = name;
         this.declineConfig = declineConfig;
         this.sirenActionAutofillParams = sirenActionAutofillParams;
-        if (testConfig) {
-            const metaCndConfig = testConfig.cnd[name];
-            if (!metaCndConfig) {
-                throw new Error("cnd configuration is needed");
-            }
-
-            this.host = metaCndConfig.host;
-
-            const configFile = metaCndConfig.config_file;
-            this.cndConfig = toml.parse(
-                fs.readFileSync(`${root}/${configFile}`, "utf8")
-            );
-        }
+        this.cndConfig = CND_CONFIGS[name];
 
         if (walletConfig) {
             this.wallet = new Wallet(name, walletConfig);
         }
     }
 
     public cndHttpApiUrl() {
-        return "http://" + this.host + ":" + this.cndConfig.http_api.port;
+        return `http://127.0.0.1:${this.cndConfig.httpApiPort}`;
     }
 
     public cndNetworkListenAddress() {
-        const addr = multiaddr(this.cndConfig.network.listen[0]);
-        // Need to convert 0.0.0.0 to 127.0.0.1
-        return `/${addr.protoNames()[0]}/${this.host}/${addr.protoNames()[1]}/${
-            addr.nodeAddress().port
-        }`;
+        return `/ip4/0.0.0.0/tcp/${this.cndConfig.comitPort}`;
```

Comment:
This function looks useless. Is it used?

Reply:
Haha, they are actually wrong :upside_down_face: 
Need to be `127.0.0.1`. Will send a follow up PR!

Reply:
Who were "they"?